frontend/src/app/queries/[userQueryId]/page.tsx

```diff
@@ -71,6 +78,7 @@ const Page: React.FC<UserQueryPageProps> = ({ params: { userQueryId } }) => {
   return (
     <div className="flex flex-col h-full">
       <QueryHeader query={userQuery} />
+      <QueryParameters parameters={parameters} setParameters={setParameters} />
```

Comment:
Would be good to merge master into this branch to see how this component interacts with the Pipelin/SQL tabs